js/src/apbct-public-bundle.js

```diff
@@ -2681,6 +2680,11 @@ function apbctProcessIframes()
 
 function apbctProcessExternalForm(currentForm, iterator, documentObject) {
 
+    //skip excluded forms
+    if ( formIsExclusion(currentForm)) {
```

Comment:
Already checked there https://github.com/CleanTalk/wordpress-antispam/blob/dbe309ce43b0867e5124ebeb47b2a09eae591470/js/src/apbct-public--5--external-forms.js#L130
Isnt it?

Reply:
You compare bundle version with part of it.